static/js/actions/ui.js

```diff
@@ -65,3 +65,18 @@ export const SET_USER_PAGE_DIALOG_VISIBILITY = 'SET_USER_PAGE_DIALOG_VISIBILITY'
 export const setUserPageDialogVisibility = bool => (
   { type: SET_USER_PAGE_DIALOG_VISIBILITY, payload: bool }
 );
+
+export const SET_SHOW_EDUCATION_DELETE_DIALOG = 'SET_SHOW_EDUCATION_DELETE_DIALOG';
```

Comment:
I think we need both for the `/users` page, we have two different popups there which each have different callbacks bound to the action buttons. I initially had just `SET_SHOW_DELETE_CONFIRM_DIALOG`, but on the `/users` page that causes a race condition between the popups for education and work history (we'll delete the entry at `DELETION_INDEX` for whichever one ends up being on top).

Does that make sense?